### DISCUSSION TITLE
Trying to match multi-part files of same movie with GEVI

### DISCUSSION THREAD

**Discussion by dimmitri9** [CodyBerenson/PGMA-Modernized#241] [Q&A] [answered]
Well, it's outstanding managing the content with this brillant work of yours, thank you!!!

I've been trying to match multi-part files with GayAdult agent (GEVI specifically)

https://gayeroticvideoindex.com/video/62379
![Screenshot 2023-05-09 233204](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/db19c85c-382a-46d7-94e0-b6754571efd2)
![Screenshot 2023-05-09 233221](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/1a9ab1c1-ee15-43b1-b3d7-6f2b5f2819ed)
![Screenshot 2023-05-09 233251](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/87b5ed41-213e-4757-aa41-9695ba924a15)
![Screenshot 2023-05-09 233315](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/41ce4588-ff69-4341-9268-ddf38420258f)
![Screenshot 2023-05-09 233324](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/39328023-061a-4189-af09-bb6bd52959bd)

and 

https://gayeroticvideoindex.com/video/30305

![Screenshot 2023-05-09 233111](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/00fd1c24-94d7-465d-a367-411ad123e196)
![Screenshot 2023-05-09 233359](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/2aced855-5e42-4d50-a508-a30b5cbfcb4c)

Already read the tutorial section about the multi-files, but nothing...

Note that I'm playing with trailers also =)

**Comment by JPH71** [comment, marked as answer]
Small error after the year you need a - followed by scene.....

(Studio) - Title (year) - Scene 1.mp4

Jason xxx

**Comment by dimmitri9** [comment]
As always, you ROCK JASON!!!!!!! 

=DDDDDDDDDDDDDD

**Comment by dimmitri9** [comment]
Just having trouble with WayBig this time
It is recognizing series name as part of the cast

https://www.waybig.com/blog/2015/11/04/luke-adams-fucks-josh-peters-in-my-moms-new-husband-part-6-at-men-com/
![Screenshot 2023-05-12 121529](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/8b0d424d-4dc8-41c2-a9af-4952328e19ea)
![Screenshot 2023-05-12 121556](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/2d1020fe-ef5b-4aa7-8caa-260c9f6021b9)
![Screenshot 2023-05-12 121604](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/5f1c8a6b-3744-4000-8558-ccdb705ac2e0)
![Screenshot 2023-05-12 193440](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/7df83144-f3bc-40e4-b7ea-91fec35dab8f)

**Comment by JPH71** [comment]
I will look into this one today and see how to sort it out....

**Reply by dimmitri9** [reply]
Have an outstanding sunday, friend

**Comment by JPH71** [comment]
Dear Dimitri,

The code to sort this out is with Cody for testing...

Next time you find an error - please raise it as an issue. Then we can keep track of it and close them as they are solved.

Have a nice week!

![image](https://github.com/CodyBerenson/PGMA-Modernized/assets/43102249/d321815d-5291-4053-9c25-ffb50a9b328f)

**Reply by dimmitri9** [reply]
Just for info, is the agent already updated? I can't seem to find it

**Comment by JPH71** [comment]
What version is on the site?
Check the last modification date in any of the utils.py file...

This should be 14th may...

I will be sending out an update today as I noticed some studio names like
Carnal+ were been added as cast.

Jason xxx

**Comment by JPH71** [comment]
The last update was on the utils.py file... so the agents version did not
go up....

**Reply by dimmitri9** [reply]
Sorry, didn't understand

**Comment by JPH71** [comment]
Each agent has a utils.py file next to the __init__.py

Everytime I change it I add a modification date to it to keep track of the
changes...

From the next version this will be reported in the log file...
